### PR TITLE
Feature/racialspells

### PR DESCRIPTION
**Pull request by stromblom**
Solves #157

### CODE REVIEW

File: src/Helpers/SpellHelper.php [hunk 1]
```diff
@@ -56,6 +57,12 @@ public function getSpells(): Collection
 
     public function getSelfSpells(): Collection
     {
+        $raceName = app(SelectorService::class)->getUserSelectedDominion()->race->name;
```
Comment (WaveHack) [inline src/Helpers/SpellHelper.php]: I don't really like that we're using the SelectorService for this. We should continue the discussion later about how to tackle this properly, imo.

File: src/Helpers/SpellHelper.php [hunk 2]
```diff
@@ -114,7 +121,37 @@ public function getSelfSpells(): Collection
...
+    public function getRacialSelfSpells(): Collection
+    {
+        return collect([
+            [
+                'name' => 'Crusade',
+                'description' => '+5% offensive power, allows you to kill Undead',
+                'key' => 'crusade',
+                'mana_cost' => 5,
+                'duration' => 12,
+                'races' => collect(['Human', 'Nomad']),
```
Comment (WaveHack) [inline src/Helpers/SpellHelper.php]: The races values here can just be a regular PHP array. I use Collections mainly so I can easily filter and search through them.

File: src/Helpers/SpellHelper.php (hunk 1 above)
Comment (stromblom) [reply src/Helpers/SpellHelper.php]: Yeah I know. We can discuss this tonight perhaps?

File: src/Helpers/SpellHelper.php (hunk 2 above)
Comment (stromblom) [reply src/Helpers/SpellHelper.php]: I'm quite the noob at what's in the framework and not. Sounds like a plan. Cheers!

File: src/Helpers/SpellHelper.php (hunk 1 above)
Comment (WaveHack) [reply src/Helpers/SpellHelper.php]: Please pass a `Race` instance into this method.

Comment (stromblom) [reply src/Helpers/SpellHelper.php]: Done!

**Pull request merged**